turbo.json

```diff
@@ -0,0 +1,48 @@
+{
+  "pipeline": {
+    "build": {
```

Comment:
You love to see it! :astonished: 
![image](https://user-images.githubusercontent.com/8978655/208941129-879932e5-2116-414b-b145-fa8a91314235.png)